azure-auth-helper/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-maven-plugins-pom</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3</version>
```

Comment:
1.0.3-SNAPSOHT?

Reply:
NO, this pom will not changed frequently